pkg/util/asyncworker/async_workers.go

```diff
@@ -259,3 +259,20 @@ func (aws *AsyncWorkers) cleanupWorkStatus() {
 		}
 	}
 }
+
+func (aws *AsyncWorkers) WorkExists(workName string) bool {
```

Comment:
it seems `AddWork` has already handled the duplicate logic, we don't need to have this judgement

Reply:
there's a little difference, the `AddWork` function will stop current work and launch the newer work but in this scenario I just want to discard the newer work when there's a running work.

Reply:
prefer to add a parameter in `AddWork` rather than do it in caller function; not stick to it though, @csfldf what do you think?

Reply:
I agree with adding a parameter in AddWork instead of judging it by the user

Reply:
get, I'll try to refine it.